components/terms_of_service/terms_of_service.test.tsx

```diff
@@ -62,29 +64,29 @@ describe('components/terms_of_service/TermsOfService', () => {
 
     test('should call updateTermsOfServiceStatus on registerUserAction', async () => {
         const wrapper = shallow(<TermsOfService {...baseProps}/>);
-        await wrapper.instance().registerUserAction(true, jest.fn());
+        await (wrapper.instance() as TermsOfService).registerUserAction(true, jest.fn());
```

Comment:
I think it might be preferable to instead add a type parameter to `shallow` instead of using `as` to cast the results of `wrapper.instance()` later on like
```
const wrapper = shallow<TermsOfService>(<TermsOfService {...baseProps}/>);
await wrapper.instance().registerUserAction(true, jest.fn());
```
I think it ends up being slightly easier if you call `wrapper.instance()` multiple times

Reply:
I was trying to do something like that, but did not realize the generic parameter was not enough.
Thanks!